src/preamble.js

```diff
@@ -930,11 +930,11 @@ function stackTrace() {
 
 // Memory management
 
-var PAGE_SIZE = 4096;
+var PAGE_SIZE = 65536; // Wasm pages are fixed to 64k.
```

Comment:
Mostly mmap, yeah, but I worry there is something else we are forgetting here.

But I agree, we can just try to go 16k everywhere, that would be nicest.